### PR TITLE
-merged from main

### PR DESCRIPTION
**Pull request by luitjansl**
-adapted river model to pass imod-python validation

### CODE REVIEW

**Review by Hofer-Julian — APPROVED** [review]

File: tests/test_metamod.py [hunk 1]
```diff
@@ -21,6 +21,52 @@ def test_lookup_table_present(metaswap_lookup_table: Path) -> None:
...
+    expected_dependencies = (
+        "fmpich2.dll",
+        "mpich2mpi.dll",
+        "mpich2nemesis.dll",
+        "TRANSOL.dll",
+    )
+
+    for dependency in expected_dependencies:
+        assert dependency in dep_dir_content
+
```
Comment (Hofer-Julian) [inline tests/test_metamod.py:68]: I think these tests have been moved, so you don't need them here again.

File: tests/test_modstrip.py [hunk 2]
```diff
@@ -49,6 +47,20 @@ def write_toml(
...
+    Fill para_sim.inp template in the folder with the path to the unsaturated
+    zone database.
+    """
+    with open(msw_folder / "para_sim_template.inp") as f:
+        para_sim_text = f.read()
+
+    para_sim_text = para_sim_text.replace("{{unsat_path}}", f"{path_unsat_dbase}\\")
+
+    with open(msw_folder / "para_sim.inp", mode="w") as f:
+        f.write(para_sim_text)
```
Comment (Hofer-Julian) [inline tests/test_modstrip.py:61]: Same here

**Review by Hofer-Julian — CHANGES_REQUESTED** [review]

File: tests/test_metamod.py (hunk 1 above)
Comment (luitjansl) [reply tests/test_metamod.py:68]: you're right- I removed them now (on main they are still here- we only moved them on the lumbricus branch)

File: tests/test_modstrip.py (hunk 2 above)
Comment (luitjansl) [reply tests/test_modstrip.py:61]: done

**Review by Hofer-Julian — APPROVED** [review]
Thank you, @luitjansl!

**Pull request merged**